app/recordtransfer/templates/recordtransfer/banner.html

```diff
@@ -1,6 +1,20 @@
-<div class="banner-container">
-    <div>
+{% load i18n %}
+<div class="banner-container flex items-center justify-between gap-6 w-full">
+    <div class="flex items-center">
         <span class="banner-text">The National Residential School Crisis Line&nbsp;</span>
         <a class="banner-link non-nav-link" href="tel:1-866-925-4419">1-866-925-4419</a>
     </div>
+    <form action="{% url 'set_language' %}" method="post" class="ml-4">
```

Comment:
A side effect of hiding the phone line is that the language picker gets moved to the left when the phone line disappears. You can fix this with this class:

```suggestion
    <form action="{% url 'set_language' %}" method="post" class="ml-auto">
```